master/buildbot/changes/p4poller.py

```diff
@@ -201,6 +201,7 @@ def _acquireTicket(self, protocol):
     def _parseTicketPassword(self, text):
         lines = text.split("\n")
         if len(lines) < 3:
+            raise P4PollerError("Failed to parse ticket in that output is less than 3 lines: {}".format(text))
             return None
```

Comment:
Do we need this return if we raise exception immediately after?

Reply:
> Do we need this return if we raise exception immediately after?

No, apparently. I will fix this.